src/controllers/ProjectUsersController.ts

```diff
@@ -0,0 +1,84 @@
+import { Request, Response } from "express";
+import { getRepository } from "typeorm";
+import { validate } from "class-validator";
+
+import { Project } from "../entities/Project";
+import { User } from "../entities/User";
+
+class ProjectUsersController {
+  static getAll = async (req: Request, res: Response) => {
+    const projectId = req.body.projectId;
+    const projectEntityRepo = getRepository(Project);
+
+    const project = await projectEntityRepo.findOneOrFail(projectId, { relations: ["users"] });
+
+    project.users = await projectEntityRepo
```

Comment:
Instead of querying again here, you could directly return `res.send(project.users)` since you are fetching the `project` with `relations: ["users"] }`.